bcgov/cas-registration
# 194 Reporting links on dashboard

Addresses bcgov/cas-reporting#194. This a way to reach report operations from the BCIERs dashboard. 

## Changes 🚧

- Update dashboard tile fixtures in backend.
- Add intermediary _reporting_ dash page for base `/reporting` route using updated tile data.

## To test 🔬

0. Run `bc_obps/make reset_db` to ensure the new fixtures are loaded. 
1. Spin up a local instance (`bc_obps/make start_db && bc_obps/make run` + `yarn dev-all`). Login with `bc-cas-dev`. In the _Reporting_ tile, click **Submit Annual Report(s)**. You should be taken to the Operations page. 
2. In the header breadcrumbs, click **> Reporting**. You should be taken to the Reporting dashboard. 
3. Click the **Submit Annual Report(s)** tile, you should be back at the Operations page
